### PR TITLE
docs: tweak docs

### PR DESCRIPTION
**Pull request by alissa-tung**
### Description 
_What behavior do you want to change, why, how does your patch achieve the changes?_

fix typo

### Testing done 
_Describe the testing strategy. Unit and integration tests are expected for any behavior changes._

### Reviewer checklist
- [x] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [x] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

File: docs/developer-guide/ksqldb-reference/table-functions.md [hunk 1]
```diff
@@ -142,4 +142,4 @@ EXPLODE(col1)
 Array
 
 This function takes an Array and outputs one value for each of the elements
-of the array. The output values have the same type as the  array elements.                                                    
\ No newline at end of file
+of the array. The output values have the same type as the  array elements.
```
Comment (JimGalasyn) [inline docs/developer-guide/ksqldb-reference/table-functions.md:145]:
```suggestion
of the array. The output values have the same type as the array elements.                                                    

```

**Review by JimGalasyn — APPROVED** [review]
Thanks for the fixes! LGTM, with one suggestion.

File: docs/developer-guide/ksqldb-reference/table-functions.md (hunk 1 above)
Comment (pathInd) [reply docs/developer-guide/ksqldb-reference/table-functions.md]: thx fixed

**Comment by JimGalasyn** [comment]
Closed in favor of https://github.com/confluentinc/ksql/pull/9148.

**Pull request closed without merging**